superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/Pinterest/Line/tooltip.ts

```diff
@@ -0,0 +1,272 @@
+        if (PERCENT_CHANGE_COLUMNS.includes(column)) {
+          const previousDate = getDateByTimeDelta[column](currentDate);
+          // eslint-disable-next-line no-param-reassign
+          accum[column] = getDataPercentChange(previousDate);
+        }
+        return accum;
+      },
+      {},
+    );
+
+    return {
+      ...percentChangeByKey,
+      [DeltaTableColumn.METRIC]: seriesName,
+      [DeltaTableColumn.VALUE]: (currentValue ?? 'null').toLocaleString(),
+    };
+  };
+
+  getDeltaTableRows(params: CallbackDataParams[], xIndex: number) {
+    const { tooltipDeltaColumns } = this.formData;
+    const deltaTableColumns = this.deltaTableColumns.filter(column => !PERCENT_CHANGE_COLUMNS.includes(column) ||  tooltipDeltaColumns.includes(column));
+    const rows = [
+      deltaTableColumns.map(column => ({
+        element: 'th',
+        style: this.getCellStyle(column),
+        data: column,
+      })),
+    ];
+    params.forEach((param) => {
+      const deltaTableData = this.getDeltaTableData(
+        param.value[xIndex],
+        param.seriesName,
+        deltaTableColumns,
+      );
+      const newRow = deltaTableColumns.map(column => {
+        const columnData = deltaTableData[column];
+        let color;
+        let data = columnData ?? '-';
+        if (column === DeltaTableColumn.METRIC) {
+          data = param.marker + escape(columnData);
+        }
+        else if (PERCENT_CHANGE_COLUMNS.includes(column) && columnData != null) {
+          data += '%';
+          if (columnData > 0) {
+            color = this.theme.colors.success.dark1;
+            data += DIRECTION_SYMBOL[DeltaDirection.UP];
+          } else if (columnData < 0) {
+            color = this.theme.colors.error.dark1;
+            data += DIRECTION_SYMBOL[DeltaDirection.DOWN];
+          }
+        }
+        return {
+          element: 'td',
+          style: this.getCellStyle(column, color),
+          data,
+        };
+      });
+      rows.push(newRow);
+    });
+    return rows;
+  }
+
+  getTooltipConfig() {
+    const { richTooltip, tooltipSortByMetric, orientation } = this.formData;
+
+    const [xIndex, yIndex] =
+      orientation === OrientationType.horizontal ? [1, 0] : [0, 1];
+
+    let tooltipConfig: TooltipOption = {
+      ...defaultTooltip,
+      appendToBody: true,
+      trigger: richTooltip ? 'axis' : 'item',
+      // eslint-disable-next-line theme-colors/no-literal-colors
+      backgroundColor: 'rgba(255, 255, 255, 0.90)',
+      formatter: (initialParams: TooltipPositionCallbackParams) => {
+        let params: CallbackDataParams[] = richTooltip
+          ? (initialParams as CallbackDataParams[])
+          : [initialParams as CallbackDataParams];
+        if (tooltipSortByMetric) {
+          params = orderBy(params, [
+            ({ value }) => -1 * value[yIndex],
+            ['desc'],
+          ]) as CallbackDataParams[];
+        }
+        const deltaTableRows = this.getDeltaTableRows(params, xIndex);
```

Comment:
even if richtooltip is false, you are using your custom formatter to display tooltip. In that case can it be defaulted to superset default tooltip formatter (or) that is not possible ?

Reply:
just saw that delta columns can be configured even for non-rich-tooltip